File: falldownview/src/main/java/com/mitsuki/falldownview/path/CircleSample.java
package com.mitsuki.falldownview.path;

import android.graphics.Path;

public class CircleSample extends ComponentSample {
    public CircleSample() {
        this.baseline = 100f;
        this.mPath = new Path();
        mPath.addCircle(baseline / 2, baseline / 2, baseline / 2, Path.Direction.CW);
    }
}

File: falldownview/src/main/java/com/mitsuki/falldownview/path/CircleSample.kt
package com.mitsuki.falldownview.path

import android.graphics.Path

class CircleSample : ComponentSample() {

    override val sizeBenchmark: Float = 20f

    override val defaultScale: Float = 0.8f

    init {
        this.mPath = Path()
        mPath.addCircle(
            sizeBenchmark / 2,
            sizeBenchmark / 2,
            sizeBenchmark / 2,
            Path.Direction.CW
        )
    }
}
